lib/documents/schemas/algorithmic_transparency_records.json

```diff
@@ -482,6 +482,6 @@
   "show_summaries": true,
   "signup_content_id": "99f5bacb-d1ef-4b93-87ef-bae129fed396",
   "subscription_list_title_prefix": "Algorithmic transparency records",
-  "summary": "<p>Find algorithmic transparency records from UK public sector organisations with information on algorithmic tools used in decision making. These are completed in accordance with the Algorithmic Transparency Recording Standard.</p><p>Let us know what you think of this service: [<a href=\"https://forms.office.com/e/XNn3GBqzhY\">feedback form</a>]</p>",
+  "summary": "Find algorithmic transparency records from UK public sector organisations with information on algorithmic tools used in decision making. These are completed in accordance with the Algorithmic Transparency Recording Standard. \r\n\r\nLet us know what you think of this service: [feedback form](https://forms.office.com/e/XNn3GBqzhY)",
```

Comment:
Good job on these 👍 I'd suggest moving all the finder changes out to a separate commit so that the `app/presenters/finder_content_item_presenter.rb` change is in its own separate commit.